server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CICSParser.g4

```diff
@@ -422,7 +422,7 @@ pf_option: PF1 | PF2 | PF3 | PF4 | PF5 | PF6 | PF7 | PF8 | PF9 | PF10 | PF11 | P
            PF18 | PF19 | PF20 | PF21 | PF22 | PF23 | PF24;
 
 /** IGNORE CONDITION */
-cics_ignore: IGNORE (CONDITION | mama | cics_resp)+;
+cics_ignore: IGNORE CONDITION? (mama | cics_resp)+;
```

Comment:
AFAIR. `CONDITION` is obligatory here. The previous implementation was incorrect.

Reply:
I'll fix it.